distribution/dnf-json.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=OSBuild dng-json service
+After=network.target
```

Comment:
I don't think you need to order after the network target, but maybe I am missing something?

Reply:
That's copy and paste from `osbuild-remote-worker@.service` and `osbuild-worker@.service` I did not think about this too much. Should I remove this ?

Reply:
I would remove it. This would make sure NetworkManager is started before the service (doesn't mean anything as the network might still be down) and stopped after it (which might be important to avoid network connection being dropped). However, we are not exposed to the network, so this should not be necessary. I would drop it. Unless someone corrects me ;) @gicmo @ondrejbudai ?

PS
I think it should not be in `osbuild-worker@.service` either, but in `-remote-` it makes sense.